pkg/sql/schemachanger/testdata/end_to_end/alter_table_alter_column_set_default/alter_table_alter_column_set_default.definition

```diff
@@ -0,0 +1,22 @@
+setup
+CREATE TABLE t (i INT PRIMARY KEY, j INT);
+ALTER TABLE t ALTER COLUMN j SET DEFAULT 2;
```

Comment:
nit: since this is setup, we could probably just do one line `create table t (i int primary key, j int default 2);`

Reply:
done